stubs/pika/pika/spec.pyi

```diff
@@ -0,0 +1,923 @@
+from _typeshed import Incomplete
+from typing import ClassVar
+from typing_extensions import TypeAlias
+
+from pika.amqp_object import Class, Method, Properties
+from pika.compat import str_or_bytes as str_or_bytes
```

Comment:
Hmm, do we need this re-export in the stubs? Or would it be better to just allowlist it? Seems like something of an implementation detail.